users/assets/css/style.css

```diff
@@ -0,0 +1,261 @@
+*,
+*:before,
+*:after  {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing:    border-box;
+    box-sizing:         border-box;
+    margin: 0;
+    padding: 0;
+}
+html {
+  height: 100%;
+}
+body {
+    position: relative;
+    margin: 0;
+    min-height: 100%;
+    font-family: 'Poppins', sans-serif;
+    background-color: #fff;
+    font-size: 16px;
+}
+h1{
+    font-size: 48px;
+}
+h2{
+    font-size: 40px;
+}
+h3{
+    font-size: 32ßpx;
```

Comment:
please check the value here of the css property

Reply:
This is fixed